### DISCUSSION TITLE
Welcome to WoeUSB-ng Discussions!

### DISCUSSION THREAD

**Discussion by WaxyMocha** [WoeUSB/WoeUSB-ng#12] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Kevlar98** [comment]
Anybody knows how to unboot a USB that was booted with windows 10 with this software?
